item-api/src/main/java/com/example/itemapi/global/config/LocalCacheConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.example.itemapi.global.config;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCache;
+import org.springframework.cache.support.SimpleCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+@EnableCaching
+public class LocalCacheConfig {
+
+    @Bean
+    public CacheManager cacheManager() {
```

Comment:
```suggestion
    public CacheManager simpleCacheManager() {
```

진짜 별것 아닌 내용이지만 
결국엔 `SimpleCacheManager` 를 응답하고 있으니 함수명도 변경되면 어떨까 하는 생각이 드네요 ㅎㅎ